packages/compass-editor/src/editor.tsx

```diff
@@ -25,25 +16,25 @@ import 'ace-builds/src-noconflict/mode-ruby';
 import 'ace-builds/src-noconflict/mode-rust';
 import 'ace-builds/src-noconflict/mode-golang';
 import 'ace-builds/src-noconflict/mode-php';
-import beautify from 'ace-builds/src-noconflict/ext-beautify';
-import 'mongodb-ace-mode';
-import '../constants/mongodb-ace-theme';
-import '../constants/mongodb-ace-theme-query';
+import './ace/mode';
+import './ace/theme';
 import tools from 'ace-builds/src-noconflict/ext-language_tools';
-import { css } from '@leafygreen-ui/emotion';
+import beautify from 'ace-builds/src-noconflict/ext-beautify';
 
 /**
  * Options for the ACE editor.
  */
 const DEFAULT_OPTIONS: IAceOptions = {
   enableLiveAutocompletion: false,
   tabSize: 2,
-  fontSize: 11,
   minLines: 10,
   maxLines: Infinity,
   showGutter: true,
   showPrintMargin: false,
   useWorker: false,
+  fontFamily: fontFamilies.code,
+  fontSize: 13,
```

Comment:
Default leafygreen font size. Needs to be passed directly to ace because some ace rendering logic relies on this number being explicitly passed to the editor